### PR TITLE
feat(navbar): make top-level items clickable on desktop

### PR DESCRIPTION
**Pull request by wescopeland**
Makes all top-level nav items clickable on desktop, except Manage.

Clicking the user avatar will take the user directly to their profile.
There is a lot of noise in this PR from `composer fix`, I think it may have been skipped on some of the prod fixes.

### CODE REVIEW

File: resources/views/components/menu/account.blade.php [hunk 1]
```diff
@@ -39,7 +39,7 @@
             <div class='softcore cursor-help' title="Points earned in softcore mode">{{ localized_number($user->RASoftcorePoints) }}</div>
         @endif
     </div>
-    <x-nav-dropdown trigger-class="py-0" dropdown-class="dropdown-menu-right">
+    <x-nav-dropdown trigger-class="py-0" dropdown-class="dropdown-menu-right" desktopHref="{{ route('user.show', $user) }}">
```
Comment (luchaos) [inline resources/views/components/menu/account.blade.php:42]: can be `:desktopHref="route('user.show', $user)"` to condense it. no strong preference.

**Review by luchaos — APPROVED** [review]

**Comment by luchaos** [comment]
> There is a lot of noise in this PR from `composer fix`, I think it may have been skipped on some of the prod fixes.

Yeah, that's because of the composer dependencies update that was introduced by a hotfix.
CS fixer does a few thing more thoroughly now.

**Comment by Jamiras** [comment]
> > There is a lot of noise in this PR from `composer fix`, I think it may have been skipped on some of the prod fixes.
> 
> Yeah, that's because of the composer dependencies update that was introduced by a hotfix. CS fixer does a few thing more thoroughly now.

I'm in the process of committing the composer fix changes directly to master

File: resources/views/components/menu/account.blade.php (hunk 1 above)
Comment (wescopeland) [reply resources/views/components/menu/account.blade.php]: Thanks, addressed in https://github.com/RetroAchievements/RAWeb/pull/1840/commits/c654adefaa3d2f62f630c9d280733fe78daa1975.

**Comment by wescopeland** [comment]
No worries, conflict was easy to resolve. I'll wait for CI and then merge.

File: resources/views/components/menu/main.blade.php [hunk 2]
```diff
@@ -140,7 +140,7 @@
     <x-dropdown-item :link="url('achievementList.php?s=4&p=2')">Easy Achievements</x-dropdown-item>
     <x-dropdown-item :link="url('achievementList.php?s=14&p=2')">Hardest Achievements</x-dropdown-item>
 </x-nav-dropdown>
-<x-nav-dropdown :title="__('Community')">
+<x-nav-dropdown :title="__('Community')" desktopHref="{{ url('forum.php') }}">
```
Comment (Jamiras) [inline resources/views/components/menu/main.blade.php:143]: I don't think anyone's yearning for the menu items to be clickable. The profile image is muscle memory from the old layout.

**Review by Jamiras — COMMENTED** [review]
Verified desktop behavior. Have no way to test the mobile changes.

File: resources/views/components/menu/main.blade.php (hunk 2 above)
Comment (wescopeland) [reply resources/views/components/menu/main.blade.php:143]: I can back this out if we prefer.

Comment (luchaos) [reply resources/views/components/menu/main.blade.php:143]: sounds good to me, wasn't sure either. profile picture should be enough for now

**Pull request merged**